.github/workflows/release.yml

```
@@ -3,7 +3,8 @@ name: release
 on:
   push:
     tags:
-      - "v*.*.*"
+      - "avalanche-types*"
```

We should be explicit to avoid human error
```suggestion
      - "avalanche-types-v*.*.*"
      - "avalanche-consensus-v*.*.*"
```

---

Agreed